ui/app/pages/create-account/connect-hardware/account-list.js

```diff
@@ -47,18 +58,24 @@ class AccountList extends Component {
   }
 
   renderHeader() {
-    const { device } = this.props;
     return (
       <div className="hw-connect">
         <h3 className="hw-connect__unlock-title">
-          {`${this.context.t('unlock')} ${this.capitalizeDevice(device)}`}
+          {this.context.t('selectAnAccount')}
         </h3>
-        {device.toLowerCase() === 'ledger' ? this.renderHdPathSelector() : null}
         <h3 className="hw-connect__hdPath__title">
           {this.context.t('selectAnAccount')}
         </h3>
         <p className="hw-connect__msg">
-          {this.context.t('selectAnAccountHelp')}
+          {this.context.t('selectAnAccountHelp', [
+            <span
+              className="hw-connect__msg__link"
+              onClick={() => this.setState({ showPopover: true })}
+              key="account-help"
+            >
+              Click here
```

Comment:
This needs to be localized

Reply:
D'oh, I added the localization to the `en` file but didn't use it.  Ugh.